sql/mob_spawn_points.sql

```diff
@@ -51631,18 +51631,18 @@ INSERT INTO `mob_spawn_points` VALUES (17367058,'GoBha_Slaughterer','Go\'Bha Sla
 INSERT INTO `mob_spawn_points` VALUES (17367059,'Gaki','Gaki',7,-177.000,60.000,-144.000,130);
 INSERT INTO `mob_spawn_points` VALUES (17367060,'Rasetsu','Rasetsu',8,-177.000,60.000,-142.000,130);
 INSERT INTO `mob_spawn_points` VALUES (17367061,'Onki','Onki',9,-177.000,60.000,-140.000,130);
-INSERT INTO `mob_spawn_points` VALUES (17367062,'Onibi','Onibi',10,1.000,1.000,1.000,130);
-INSERT INTO `mob_spawn_points` VALUES (17367063,'Ayakashi','Ayakashi',11,0.000,0.000,0.000,0);
+INSERT INTO `mob_spawn_points` VALUES (17367062,'Onibi','Onibi',10,-177.000,60.000,-140.000,130);
+INSERT INTO `mob_spawn_points` VALUES (17367063,'Ayakashi','Ayakashi',11,-177.000,60.000,-140.000,130);
 INSERT INTO `mob_spawn_points` VALUES (17367064,'Gaki','Gaki',7,23.274,0.000,15.964,130);
 INSERT INTO `mob_spawn_points` VALUES (17367065,'Rasetsu','Rasetsu',8,23.274,0.000,17.964,130);
 INSERT INTO `mob_spawn_points` VALUES (17367066,'Onki','Onki',9,23.274,0.000,19.964,130);
-INSERT INTO `mob_spawn_points` VALUES (17367067,'Onibi','Onibi',10,201.274,-59.000,160.964,130);
-INSERT INTO `mob_spawn_points` VALUES (17367068,'Ayakashi','Ayakashi',11,0.000,0.000,0.000,0);
+INSERT INTO `mob_spawn_points` VALUES (17367067,'Onibi','Onibi',10,23.274,0.000,19.964,130);
```

Comment:
Your PR seems to mention only battlefield 1, but it looks like here all of them are missing entries? Are there any reference caps?

Reply:
No I just noticed they were zero but we're listed mobids in the battlefield

These are the avatars for onki, they only come out when he uses astral flow which is I assume why they didn't have positions